### PR TITLE
Symlink data to `examples/` so all examples can be easily open in oth…

### PR DESCRIPTION
**Pull request by kanitw**
…er applications such as Vega Desktop and Voyager

### CODE REVIEW

**Comment by domoritz** [comment]
Have you tried running the website? I just want to make sure that Jekyll doesn't choke on it.

**Comment by kanitw** [comment]
Hmm weird, even though node_modules is in the exclude list, I still see this error:

```
Incremental build: disabled. Enable with --incremental
      Generating...
                    done in 8.154 seconds.
        ** ERROR: directory is already being watched! **

        Directory: /Users/kanitw/Documents/_code/_idl/_visrec/vega-lite/examples/specs/data

        is already being watched through: /Users/kanitw/Documents/_code/_idl/_visrec/vega-lite/node_modules/vega-datasets/data

        MORE INFO: https://github.com/guard/listen/wiki/Duplicate-directory-errors
 Auto-regeneration: enabled for '/Users/kanitw/Documents/_code/_idl/_visrec/vega-lite'
Configuration file: /Users/kanitw/Documents/_code/_idl/_visrec/vega-lite/_config.yml
        ** ERROR: directory is already being watched! **

        Directory: /Users/kanitw/Documents/_code/_idl/_visrec/vega-lite/node_modules/vega-datasets/data

        is already being watched through: /Users/kanitw/Documents/_code/_idl/_visrec/vega-lite/node_modules/vega-datasets/data
```

**Comment by kanitw** [comment]
But the site is still running fine though

**Comment by domoritz** [comment]
Do we have to manually create the symlink when we build the examples because the symlink gets deleted? Can you also move the normalized examples?

Other than that. LGTM.

**Comment by kanitw** [comment]
Pushed update to script to remove generated specs, not the whole folder.

Move normalized spec is a separate sub-issue.  This PR and the squash commit should only deal with sym-linking.

File: scripts/build-examples.sh [hunk 1]
```diff
@@ -5,8 +5,9 @@ dir=${dir-"examples/compiled"}
 
 echo "compiling examples to $dir"
 
-rm -rf $dir
-mkdir $dir
+mkdir -p $dir
```
Comment (domoritz) [inline scripts/build-examples.sh]: The mkdir is not necessary because we never delete the directory

Comment (kanitw) [inline scripts/build-examples.sh]: Good catch -- fixed!

**Pull request merged**